#! /bin/sh /usr/share/dpatch/dpatch-run
## 10_cupsd.conf2.dpatch by Kenshi Muto <kmuto@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Debian specific patch (Listen and Browsing are controlled by debconf)

@DPATCH@
diff -urNad cupsys-1.1.99.b1.r4748~/conf/cupsd.conf.in cupsys-1.1.99.b1.r4748/conf/cupsd.conf.in
--- cupsys-1.1.99.b1.r4748~/conf/cupsd.conf.in	2005-11-13 05:24:18.352183397 +0000
+++ cupsys-1.1.99.b1.r4748/conf/cupsd.conf.in	2005-11-13 05:25:06.057880979 +0000
@@ -18,7 +18,7 @@
 @CUPS_LISTEN_DOMAINSOCKET@
 
 # Show shared printers on the local network.
-Browsing On
+#Browsing On
 BrowseOrder allow,deny
 BrowseAllow @LOCAL
 
